package.json

```diff
@@ -595,6 +595,15 @@
                     "default": false,
                     "description": "Enables remote debugging of a Web App. This experimental preview only supports node on Linux."
                 },
+                "appService.deploymentConfigurations": {
+                    "scope": "resource",
+                    "type": {
+                        "app": "string",
+                        "subpath": "string",
```

Comment:
I think you should use the existing subpath setting. I think it's confusing to have two separate settings